decidim-dev/lib/decidim/test/base_spec_helper.rb

```diff
@@ -42,3 +42,7 @@
 end
 
 require_relative "i18n_spec"
+
+def translated(field, locale: I18n.locale)
```

Comment:
This should be put into a module and included in `RSpec::Helpers`: https://www.relishapp.com/rspec/rspec-core/docs/helper-methods/define-helper-methods-in-a-module#include-a-module-in-all-example-groups